lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -2622,6 +2622,11 @@ void __cdecl _alloca_probe_16()
                 // Caller of this function expects nullValue from the requestContext.
                 funcCaller = nullValue;
             }
+            if (GeneratorVirtualScriptFunction::Is(funcCaller))
```

Comment:
You don't need to create a `GeneratorVirtualScriptFunction::Is()` method, instead you can just test that `funcCaller` is a `ScriptFunction` via `funcCaller->IsScriptFunction()`.  `GetRealFunctionObject` is provided on `ScriptFunction` and returns itself if it is not a `GeneratorVirtualScriptFunction`.